### ISSUE TITLE
Workflowmanagement und Postausgang [FEATURE]

### ISSUE THREAD

**Issue by GuidoBo** [jlawyerorg/j-lawyer-org#1073]
Wir würden uns dringend einen Workflow für Schreiben wünschen:
**Korrigieren** -> **Ausfertigen** bzw. **Signieren**-> **Versand (beA)** oder **Versand (Post)** oder **Versandt (FAX)** -> **Postbuch**
 
Der Hintergrund ist folgender:

Mitarbeitende bereiten das Schreiben vor und legt es in die Korrektur. Die Anwältin macht finale Korrekturen und legt es in die Ausfertigung. In der Ausfertigung wir signiert, sofern es eine beA-Nachricht ist oder Ausgedruckt, sofern es ein Brief ist. Der Versand wird dann in Versandt von der Zuständigen Person getätigt bzw. bestätigt und schließlich wird das Dokument im Postbuch als verschickt markiert (Versandweg, Datum, Mitarbeitende Person).

Auf diesem Weg kann von mehreren Personen vorbereitet und gesammelt werden. Es wird eine analoge Postmappe nachgebildet.

**Comment by j-dimension** [comment]
Man kann dort mit der Nutzung der Dokumentetiketten schon ziemlich weit kommen. 

In den Einstellungen bspw. Dokumentetiketten "korrigieren MA1", "versenden beA MA2" etc. konfigurieren und die Etiketten je nach Bearbeitungsstand / "Position" im Workflow aktivieren oder deaktivieren. 

Die einzelnen MA abonnieren sich dann die für sie relevanten Etiketten auf dem Desktop und haben so ihren individuellen Arbeitsvorrat.

**Comment by GuidoBo** [comment]
Ja, das haben wir gefunden, aber es ist aus unserer Sicht nicht so schon. Die MA sind in den Workflow von Advolux verliebt.

**Comment by GuidoBo** [comment]
Als Workaround wäre es vielleicht möglich, die Ansicht auf dem Desktop zu personalisieren, sodass man eine Anzeige für Etikett 1 und eine für "Etikett 2" hat?!

**Comment by j-dimension** [comment]
Wie sind die Workflows bei Advolux denn umgesetzt? Ich kenne die Anwendung nur vom Namen her.

**Comment by GuidoBo** [comment]
Das ist der Workflow von Advolux. 
- Links: eine Liste der Dokumente im Workflow. Die Dropdown-Liste ("nächster Schritt") bestimmt den weiteren Verlauf für jedes einzelne Dokument. 
- Unter Ausfertigung werden insbesondere die Anlagen eingefügt. 
- Gedruckte Post geht den unteren Weg nach (Brief).
Unten Rechts (Ausführen) führt für alle verfügten Dokumente die jeweilige Verfügung aus.
Oben links kann man die Ansicht einschränken (meine Dokumente, nur Vertretungsdokumente, Alle Dokumente)

![Advolux_Workflow](https://user-images.githubusercontent.com/56734761/104852115-1eb1c580-58f9-11eb-9ace-09d8f9141d78.png)

**Comment by j-dimension** [comment]
Kann man die selbst modellieren oder sind sie von der Anwendung vorgegeben?

**Comment by GuidoBo** [comment]
Die sind komplett vorgegeben.

**Comment by j-dimension** [comment]
Danke. Ich empfinde das durchaus als ein richtig gutes Feature, das man an vielen Stellen verwenden kann. Problem ist der Umsetzungsaufwand. Ich packe das jetzt mal in 2.0. 

Das nächste "große" Feature müssen erstmal die Finanzen sein, dann schaue ich auf die nächsten großen Punkte auf der Roadmap.

**Comment by GuidoBo** [comment]
Für den Moment würde es wohl genügen, wenn man eine Ansicht (ähnlich dem Desktop) hätte, die bestimmte Etiketten nebeneinander zeigt (ähnlich einem Kanban-Board). Das wäre schon mal ein riesen Gewinn für uns, denn so hat man sofort einen Überblick, was aktuell auf Freigabe wartet / zu korrigieren ist / noch raus muss.

**Comment by j-dimension** [comment]
> Für den Moment würde es wohl genügen, wenn man eine Ansicht (ähnlich dem Desktop) hätte, die bestimmte Etiketten nebeneinander zeigt

Dafür gibt es ein separates Ticket #981 
Konkrete Umsetzungsvorschläge am besten dort als Kommentar hinterlassen.
